go/vt/mysqlctl/mysqld.go

```diff
@@ -518,16 +532,23 @@ func (mysqld *Mysqld) ExecuteMysqlCommand(sql string) error {
 	return nil
 }
 
-// GetDbaConnection returns a connection from the dba pool.
+// GetDbConnection returns a connection from the pool chosen by dbconfigName.
 // Recycle needs to be called on the result.
-func (mysqld *Mysqld) GetDbaConnection() (dbconnpool.PoolConnection, error) {
-	return mysqld.dbaPool.Get(0)
+func (mysqld *Mysqld) GetDbConnection(dbconfigName string) (dbconnpool.PoolConnection, error) {
+	switch dbconfigName {
```

Comment:
I was thinking of creating:
type DbConfigName string
then:
const DbaConfigName DbConfigName = "dba"
const AppConfigName DbConfigName = "app"
and use that in the signatures / switch statements everywhere...